<launch>

	<node pkg="course_agv_icp" type="icp" name="icp" output="screen">
	<!--实现后，course_agv_icp改为python文件-->
	<!-- <node pkg="course_agv_icp" type="odometry.py" name="icp" output="screen"> -->
		<!--from a rosbag-->
		<param name="robot_x" value="0"/>
		<param name="robot_y" value="0"/>
		<param name="robot_theta" value="0"/>

		<param name="max_iter" value="50"/>
		<param name="tolerance" value="0"/>
		<param name="dis_th" value="0.05"/>
	</node>
	<!--tf-->
    <node pkg="course_agv_icp" type="tf" name="my_tf"/>
	<!--rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_icp)/launch/course_agv.rviz" />
</launch>
